docs/REST API/rest-api.md

```diff
@@ -4584,6 +4592,33 @@ For example, a policy might ask “Is Gatekeeper enabled on macOS devices?“ Th
 }
 ```
 
+---
+
+### Count Policies
+
+`GET /api/v1/fleet/global/policies/count`
+
+#### Example
+
+`GET /api/v1/fleet/global/policies/count`
+
+#### Parameters
+| Name               | Type    | In   | Description                                                                                                   |
+| ------------------ | ------- | ---- | ------------------------------------------------------------------------------------------------------------- |
+| query                 | string | query | Search query keywords. Searchable fields include name. 
```

Comment:
Few changes here:
+ By convention this should go above "Example"
+ Let's wrap `name` in backticks to match the other examples
+ Not 100% sure it matters,  but would close the table row just in case it causes issues building the docs from markdown
```suggestion
| query                 | string | query | Search query keywords. Searchable fields include `name`.  |
```